fe/fe-core/src/main/java/com/starrocks/privilege/DefaultAuthorizationProvider.java

```diff
@@ -33,88 +33,88 @@ public class DefaultAuthorizationProvider implements AuthorizationProvider {
     private static final short PLUGIN_ID = 1;
     private static final short PLUGIN_VERSION = 1;
 
-    private static final Map<ObjectType, List<PrivilegeType>> TYPE_TO_ACTION_LIST =
```

Comment:
‌‌‌‌Why do we need to change the static class object TYPE_TO_ACTION_LIST to typeToActionList?

Reply:
Because some new privilegeTypes need to be added in subsequent logic